files/sinonjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "sinon",
+  "repo": "sinonjs/sinon",
+  "files": {
+    "basePath": "lib/",
+    "include": ["sinon.js"]
```

Comment:
Awesome. I'll submit the issue now. What should i do about this issue in the meantime?